src/node_v8.cc

```diff
@@ -45,6 +63,24 @@ void UpdateHeapStatisticsArrayBuffer(const FunctionCallbackInfo<Value>& args) {
 }
 
 
+void UpdateHeapSpaceStatisticsBuffer(const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+  HeapSpaceStatistics s;
+  Isolate* const isolate = env->isolate();
+  uint32_t* buffer = env->heap_space_statistics_buffer();
+
+  for (size_t i = 0; i < number_of_heap_spaces; i++) {
+    isolate->GetHeapSpaceStatistics(&s, i);
+    size_t const propertyOffset = i * kHeapSpaceStatisticsPropertiesCount;
+    #define V(index, name, _) buffer[propertyOffset + index] = \
```

Comment:
Two style nits: can you use `snake_case` instead of `camelCase` for locals in C++?  And can you not indent the `#define` keyword?